nilearn/_utils/ndimage.py

```diff
@@ -6,12 +6,12 @@
 
 import numpy as np
 from scipy import ndimage
-
-
```

Comment:
I see the point, won't do this in the future... I got a bit overzealous when I got flake working.